modules/manage/pages/manage-security/configure-saslauthd.adoc

```diff
@@ -19,11 +19,46 @@ The `saslauthd` agent, which must be installed and configured on each Couchbase
 Note that _either_ Native LDAP Support _or_ `saslauthd` must be selected for the cluster.
 The two cannot be used simultaneously.
 
-For an overview of authentication option on Couchbase Server, see xref:learn:security/authentication-overview.adoc[Authentication].
+For an overview of authentication options on Couchbase Server, see xref:learn:security/authentication-overview.adoc[Authentication].
+
+[#migrating-from-saslauthd-to-native-ldap]
+=== Migrating from saslauthd to Native LDAP
+
+If you have been using `saslauthd` with previous versions of Couchbase Server, you are strongly recommended to migrate to _Native LDAP_.
+See xref:learn:security/authentication-overview.adoc[Authentication] for a full account of the benefits.
+
+To migrate, proceed as follows:
+
+. Configure Native LDAP.
+This is described in detail, in xref:manage:manage-security/configure-ldap.adoc[Configure LDAP.]
++
+Note that the procedure provides two principal options, which are *Enable LDAP user authentication* and *Enable LDAP group authentication & sync*.
+During your configuration procedure:
+
+** You _must_ select *Enable LDAP user authentication*, and then configure associated details.
+
+** If LDAP group authentication is required, you _can_ select *Enable LDAP group authentication & sync*, and then configure associated details.
+Alternatively, you can leave this option disabled for now: it does not need to have been enabled, for migration to succeed.
+Instead, it can be enabled and configured later, after migration has been completed.
+
+. Disable `saslauthd` authentication, as follows:
++
+----
+couchbase-cli setting-saslauthd -c 127.0.0.1:9001 -u Administrator -p password --enabled 0
```

Comment:
please replace "-c 127.0.0.1:9001" with something that usually is used in examples. I see that "-c 10.143.192.101" is used in example on the same page.

Reply:
Done.

Reply:
I've also added a section and link as /manage/manage-security/configure-ldap.html#migrating-to-native-ldap, and have added a link to /learn/security/authentication-overview.html#saslauthd-and-pam